src/GraphQlSdk.php

```diff
@@ -197,6 +197,60 @@ public function getMethods(string $cartTotal = null, string $currency = null, ar
                                 }
                             }
                         }
+                    ... on ClearpayPaymentMethodSettings {
+                            enableWhenConfigured
+                            checkout {
+                                theme {
+                                    name
```

Comment:
I think you can just request the value field as thats all you need on the plugin. Similar to paypal
```
... on PaypalPaymentMethodSettings {
                            checkout {
                                button {
                                    enabled
                                    layout {
                                        value
                                    }
                                    color {
                                        value
                                    }
                                    shape {
                                        value
                                    }
                                    label {
                                        value
                                    }
                                    tagline
                                    size
                                }
```